opentelemetry/src/context.rs

```diff
@@ -460,6 +460,15 @@ impl ContextStack {
             // The empty context is always at the bottom of the [`ContextStack`]
             // and cannot be popped, and the overflow position is invalid, so do
             // nothing.
+            otel_warn!(
+                name: "Context.OutOfOrderDrop",
+                position = pos,
+                message = if pos == ContextStack::BASE_POS {
+                    "Attempted to pop the base context which is not allowed"
```

Comment:
I don't believe codecov is right; the test I added with out of order async drops hits this line in the debugger. Weird.

Reply:
I think the issue is that the tests run with the logging macros evaluating to nothing, so there is nothing to run there.

Reply:
oooh good catch.
I've jazzed up the macros so they print to stdout in test. You need to `--nocapture` cargo test to see this, so by default it goes to /dev/null unless you want to see it.

Reply:
... and as predicted this fixes the coverage lint.